Makefile

```diff
@@ -114,7 +114,7 @@ new_env: clean
 	then\
 		pipenv --rm;\
 		pipenv --clear;\
-		pipenv --python 3.10;\
+		pipenv --python 3.11;\
```

Comment:
not sure, is it good. does majority of devs have 3.11.

it can bring some issues on the older systems with no python 3.11 or need to use pyenv